src/components/sections/home/main-features/main-features.module.scss

```diff
@@ -37,7 +37,7 @@
 
 @media (min-width: $screen-md) {
   .section {
-    margin-top: 200px;
+    margin-top: -220px;
```

Comment:
Instead of setting this margin in breakpoints let's set it in a class that is added when the scroll trigger is created. Bc we are creating the scroll trigger based on DeviceType instead of window size.

Reply:
Otherwise, this happens
 
<img width="855" alt="image" src="https://user-images.githubusercontent.com/43894343/186182693-5b818e6e-8c16-484a-9ff9-e5495d71c645.png">

Reply:
Agregado solamente en el useEffect que genera el ScrollTrigger!